packages/components/src/Card/CardClickable.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode, createRef } from "react";
+
+interface ClickableCardProps {
+  onClick(event: React.MouseEvent<HTMLElement>): void;
+  className: string;
+  readonly children: ReactNode | ReactNode[];
+}
+
+const ENTER_KEY = "Enter";
+const SPACEBAR_KEY = " ";
+
+/**
+ * This is only intended to be used in the Card component.
+ * Please use `<Card onClick={onClick} />` component instead.
+ */
+export function CardClickable({
+  className,
+  onClick,
+  children,
+}: ClickableCardProps) {
+  const cardRef = createRef<HTMLDivElement>();
+
+  return (
+    <div
+      ref={cardRef}
+      data-testid="clickable-card"
+      className={className}
+      onClick={onClick}
+      onKeyUp={handleKeyup}
+      onKeyDown={handleKeyDown}
+      role="button"
+      tabIndex={0}
+    >
+      {children}
+    </div>
+  );
+
+  function handleKeyup(event: React.KeyboardEvent<HTMLDivElement>) {
+    if (event.key === ENTER_KEY || event.key === SPACEBAR_KEY) {
+      cardRef.current?.click();
```

Comment:
Went with manually triggering a click instead of passing in the event since the `KeyboardEvent` is different from a `MouseEvent`. Also, because typescript.

Reply:
I wonder if card content could ever have an anchor tag or a button? 
If they do, those actions won't be triggered.

e.g. Jobber link will not work 
![image](https://user-images.githubusercontent.com/84454304/132043621-65c62322-4477-47f9-864e-cd77defd83f5.png)

Reply:
I doubt we'd do that but it could happen. I could also make sure that the keyup listeners only fire when the focused element is actually the card and not things inside it. good catch!

Reply:
Awesome Cheers 🍻

Reply:
So, new learnings today. The native enter press on a link also fires a click which bubbles up to the parent. So we can't stop it. haha

Reply:
So, it will not fire the enter/space keyup's anymore unless you're focused on the card itself. it firing the callback when focused on the link and pressing enter is a native thing that i think we should not remove.